lib/paperclip/storage/azure.rb

```diff
@@ -245,32 +220,18 @@ def flush_writes #:nodoc:
         @queued_for_write = {}
       end
 
-      def save_blob(container_name, storage_path, file, write_options)
-
-        if file.size < 64.megabytes
-          azure_interface.create_block_blob container_name, storage_path, file.read, write_options
-        else
-          blocks = []; count = 0
-          while data = file.read(4.megabytes)
-            block_id = "block_#{(count += 1).to_s.rjust(5, '0')}"
-
-            azure_interface.put_blob_block container_name, storage_path, block_id, data
-
-            blocks << [block_id]
-          end
-
-          azure_interface.commit_blob_blocks container_name, storage_path, blocks
-        end
+      def save_blob(storage_path, file, write_options)
```

Comment:
Hm, guess this wasn't really part of the public interface despite not having the `:nodoc:` tag.

Reply:
Yeah :face_with_diagonal_mouth:  this one is a breaking change. It should not matter for mastodon, but I think it's worth changing major version when cutting the new release.